README.md

```diff
@@ -161,7 +162,13 @@ See the later topic, *Install Intel(R) SGX PSW*, for information on how to insta
   $ make
   $ ./app
 ```
-   Use similar commands for other code samples.   
+   Use similar commands for other code samples.
+
+   NOTE: You may need to set up the needed environment variables before compiling the code sample. To do so, run:
```

Comment:
In the main readme, we can mention the note in the "Install the Intel(R) SGX SDK" step, because it is not only required by sampe code, but also other enclaves.